javascript/script.js

```diff
@@ -8,18 +8,19 @@ let newItemInput;
 
 const $ = id => document.getElementById(id);
 const setToDoList = list => toDoList = list;
-
+let searchInput = $("search-text");
+const validate = () => {
+	//modal display on empty input
+	if (newItemInput.value.trim() === "") {
+		showEmptyInputModal();
+		return true;
```

Comment:
The function is supposed to return the result of validation as the name suggests. So when validation fails, then it doesn't make sense to return `true`.